src/NuGet.Core/NuGet.Packaging.Core/FingerprintInfos.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System;
+
+namespace NuGet.Packaging.Core
+{
+    public class FingerprintInfos
+    {
+        private IDictionary<string, string> _keyValuePairs;
+
+        public FingerprintInfos(IDictionary<string, string> fingerPrints)
+        {
+            _keyValuePairs = fingerPrints ?? throw new ArgumentNullException(nameof(fingerPrints));
+        }
+
+        // Get fingerprint from hash algorithm oid.
+        public object this[string key]
```

Comment:
Why is this returning `object`?  It should be `string`.  `_keyValuePairs` is defined as `IDictionary<string, string>`.